glide.yaml

```diff
@@ -0,0 +1,14 @@
+package: github.com/yunify/qingcloud-sdk-go
+import:
+- package: github.com/DATA-DOG/godog
```

Comment:
Test dependency should be put in a separated glide file, and the `vendor` directory can be ignored.

Refer: https://github.com/yunify/qingstor-sdk-go/tree/master/test